src/main/java/com/chapter4/Employee.java

```diff
@@ -0,0 +1,69 @@
+package com.chapter4;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.Scanner;
+
+public class Employee {
+    private String firstName;
+    private String lastName;
+    private LocalDate birthDate = LocalDate.ofEpochDay(0);
+    private final LocalDate hiringDate;
+    private LocalDate nearestSalaryDate;
+    private static int employeeId = 0;
+    private static final LocalDate now = LocalDate.now();
+
+    public Employee(String first, String last) {
+        firstName = first;
+        lastName = last;
+        hiringDate = now;
+        employeeId++;
+        System.out.printf("Employee #%s: %s, %s, %s, %s.\n", employeeId, firstName, lastName, birthDate.toString(), hiringDate.toString());
+    }
+
+    public Employee(String first, String last, String birth) {
```

Comment:
LocalDate is better to path than String as than object is valid all the time, you are safe